### DISCUSSION TITLE
Subordinate clauses incorrectly parsed by displacy

### DISCUSSION THREAD

**Discussion by pipkin45** [explosion/spaCy#11397] [Help: Model Advice]
The following are problematic sentences for dependency parsing (English-en_core_web_sm(v3.40))
That she refused the flowers surprised him. - not correct
That she refused surprised him. - correct.
The books, which you ordered, arrived. - correct
The books which you ordered arrived. -  not correct

What can be done to improve accuracy? 
larger model?
retraining parser with Prodigy?

Thanks,
pipkin45

**Comment by polm** [comment]
If you aren't happy with the output of a model, generally the best way to get better accuracy is use more training data. In this case you might want to try larger models.
